### PR TITLE
Emphasize nearby groups

### PR DESCRIPTION
**Pull request by nicksellen**
Backend PR: ~~https://github.com/yunity/karrot-backend/pull/1065~~ merged
Next backend PR: https://github.com/yunity/karrot-backend/pull/1072

## What does this PR do?

Emphasizes nearby groups in the group gallery map. The list will order by distance (or member count if distance is not available). It will emphasis groups that are within 3x the distance of the nearest group to you, e.g.: if the nearest group is 50km -> will emphasis groups within 150km. It has a minimum "nearest group" value of 30km (to avoid scenarios like: nearest group is 1km -> will emphasis all groups within 3km = not enough groups!). Arguably this "30km" value could be less?

Might need some refinement when trying it out with the real group locations, or maybe some better sample data...

It also removes special handling for the playground group now we are de-emphasizing that feature.

Branch deployment --> https://change-emphasise-nearby-groups.dev.karrot.world

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] no unrelated changes
- [ ] asked someone for a code review
- [ ] joined [chat.foodsaving.world/channel/karrot-dev](https://chat.foodsaving.world/channel/karrot-dev)
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))

### CODE REVIEW

File: src/boot/loadInitialData.js
Comment (tiltec) [inline src/boot/loadInitialData.js:6]: can be removed

File: src/boot/loadInitialData.js
Comment (tiltec) [inline src/boot/loadInitialData.js:10]: comment can be removed

**Review by tiltec — COMMENTED** [review]
Thanks for working on this! Seems a nice approach!

I wonder how it works when no group is close to the users location? Probably it zooms out quite a lot, possibly showing half of the world?

Also, maybe it would be good to add a small comment onto the group list that it's sorted by distance?

**Comment by nicksellen** [comment]
> I wonder how it works when no group is close to the users location? Probably it zooms out quite a lot, possibly showing half of the world?

Yes, it _should_ revert to existing behaviour. Not because it's great, but to just try and constrain this PR to thinking about the new behaviour and not do a wider review of other scenarios.

I updated the top comment with an updated description of the current behaviour, and wondered if the 30km should be reduced, to 10/15/20km...

_or_ we just get this merged, and deployed, and assess it with more real scenarios and iterate on that... I think it can't be much _worse_ than current behaviour!

> Also, maybe it would be good to add a small comment onto the group list that it's sorted by distance?

I _think_ what might be better is to actually seperate it into two sections, "nearby groups", and other groups. Actually a bit like [this idea](https://github.com/yunity/karrot-frontend/issues/1704#issuecomment-526868848) from @brnsolikyl :+1: 

> The group tiles shown first are "Groups near you" and scrolling down or zooming out you'd have the rest on "Discover the world".

But I think I'd prefer to do that in future work, or support @brnsolikyl to do it maybe :)

**Comment by brnsolikyl** [comment]
Yeah, I think that to fine-tune it, we would have to play around with real-world scenarios, so I'm positive of merging it the way it is now.

I'd be happy to give a shot at the frontend later, @nicksellen !

**Comment by nicksellen** [comment]
Great! I'll wait for an approval to be checked then go!

**Review by tiltec — APPROVED** [review]

**Pull request merged**
